Hackaton2023IOS.xcodeproj/project.pbxproj

```diff
@@ -237,6 +242,7 @@
 				B26CDDAA2A0FD79000F4177D /* SectionInfoBuilder.swift */,
 				CA8464322A0FCB0C00103835 /* ProfileTableView.swift */,
 				B26CDDAF2A0FD9C700F4177D /* ProfileViewControllerTableModel.swift */,
+				CA9F3E072A10FB0D00C93E90 /* ProfileSubCategoryTableViewCell.swift */,
```

Comment:
Subcategory is one word, please rename ProfileSubcategoryTableViewCell.swift

Reply:
Also check other places

Reply:
Done